connection_test.go

```diff
@@ -291,8 +291,9 @@ func TestServiceName(t *testing.T) {
 
 	expectServiceName := serviceNameStub
 	for i := 0; i < 5; i++ {
-		sc.exec(context.Background(), "", false, /* noResult */
+		_, err := sc.exec(context.Background(), "", false, /* noResult */
```

Comment:
The comma should probably be after the comment

Reply:
Fixed.

Reply:
I don't think it is